vite.config.ts

```diff
@@ -1,10 +1,14 @@
-import { defineConfig } from "vite";
-import react from "@vitejs/plugin-react";
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
+import tsconfigPaths from 'vite-tsconfig-paths';
 
 // https://vitejs.dev/config/
 export default defineConfig({
   optimizeDeps: {
-    include: ["@endo/init"],
+    include: ['@endo/init'],
+  },
+  plugins: [react(), tsconfigPaths()],
+  build: {
+    target: 'es2020',
```

Comment:
does this mean the build app only works where es2020 does? What browsers?

Reply:
Yea, if I don't have this then `yarn build` fails complaining about BigInt literals such as `100n`.

Reply:
K. I guess we're already only supporting Chromium-based and [Chrome has had bigint since 40 versions back](https://caniuse.com/bigint).

Reply:
Hmm yea looks like the default includes safari13 which seems incompatible in that table:
```
Big integer literals are not available in the configured target environment ("chrome87", "edge88", "es2020", "firefox78", "safari13" + 2 overrides)
```